client/containers/Pub/PubDocument/PubDiscussions/Discussion/Discussion.tsx

```diff
@@ -30,7 +30,7 @@ type Props = {
 	searchTerm?: string;
 };
 
-const sortThreadComments = (threadComments: ThreadComment[], sortType: SortType) => {
+const sortThreadComments = (threadComments: ThreadCommentType[], sortType: SortType) => {
```

Comment:
Thoughts on using a common prefix to never have type aliases clash with variable names? Like `T` or `$`? I'm not strongly opinionated, maybe something to discuss in a dev huddle.

Reply:
This happens a ton on the server where TS definitions collide with model singletons. I've started to just:

```ts
import * as types from 'types`
```

and then refer to `types.CollectionPub`, reserving `CollectionPub` for the model itself. I didn't do that here because this `ThreadCommentType` is something weird and one-off.

I don't have strong opinions about prefixes, either. For readability I think `$` is better than `T`, but the meaning is less clear at a glance.

Reply:
Cool, I like the `import *` trick, so I'll start doing that too 👍